static/jobs_table.html

```diff
@@ -46,6 +46,7 @@
 	  <option value="SEMAPHORED">SEMAPHORED</option>
 	  <option value="READY">READY</option>
 	  <option value="DONE">DONE</option>
+ 	  <option value="PASSED_ON">PASSED_ON</option>
```

Comment:
Can the display value be changed to PASSED < 1 space > ON?

Reply:
In the interface (whether GuiHive, or scripts), we've tended to stick with actual database values and formatting, e.g. all caps, using underscores, etc.